documentation/modules/proc-enabling-tracing-in-connect-mirror-maker-resources.adoc

```diff
@@ -51,15 +51,15 @@ metadata:
 spec:
   #...
   template:
-    mirrorMakerContainer: <1>
+    mirrorMakerContainer:
       env:
         - name: JAEGER_SERVICE_NAME
           value: my-jaeger-service
         - name: JAEGER_AGENT_HOST
           value: jaeger-agent-name
         - name: JAEGER_AGENT_PORT
           value: "6831"
-  tracing: <2>
+  tracing:
```

Comment:
I'm guessing you didn't mean to remove these callouts, since they're still referenced below.

Reply:
The callouts below relate to the two code snippets above it. It was not rendering correctly downstream, using 2 x <1> and <2>, so the second lot were removed. I think it makes sense like that still. the alternative was to duplicate the callouts for each snippet. I tried it but preferred it this way.